spec/lib/state_spec.rb

```diff
@@ -22,12 +22,34 @@
 
   context 'a deploy to production in progress' do
     it 'reports the correct data' do
-      VCR.use_cassette("deploy-to-production-in-progress") do
+      VCR.use_cassette('deploy-to-production-in-progress') do
```

Comment:
Do we rubocop? Maybe we should

Reply:
yes good call, I think a separate PR?

![image](https://user-images.githubusercontent.com/450843/75770948-df11d780-5d40-11ea-8028-4dc70b4c9ece.png)